pkgs/development/python-modules/hvplot/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, bokeh
+, holoviews
+, pandas
+, pytest
+, parameterized
+, nbsmoke
+, flake8
+, coveralls
+, xarray
+, networkx
+, streamz
+}:
+
+buildPythonPackage rec {
+  pname = "hvplot";
+  version = "0.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "bce169cf2d1b3ff9ce607d1787f608758e72a498434eaa2bece31eea1f51963a";
+  };
+
+  checkInputs = [ pytest parameterized nbsmoke flake8 coveralls xarray networkx streamz ];
+  propagatedBuildInputs = [
+    bokeh
+    holoviews
+    pandas
+  ];
+
+  checkPhase = ''
+    HOME=$(mktemp -d) python setup.py test
```

```suggestion
    HOME=$(mktemp -d) ${python.interpreter} setup.py test
```
However, exporting `$HOME` in `preCheck` is better.

---

Fixed.